submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionApiTest.kt

```diff
@@ -247,6 +249,38 @@ class SubmissionApiTest(
         assertThat(expectedFile).hasContent("16-9 file content")
     }
 
+    @Nested
+    @SpringBootTest(webEnvironment = RANDOM_PORT, properties = ["app.subBasePath=base/path"])
+    inner class SubmitWebBasePath(@LocalServerPort val serverPort: Int) {
+        private lateinit var webClient: BioWebClient
+
+        @BeforeAll
+        fun init() {
+            webClient = getWebClient(serverPort, SuperUser)
+        }
+
+        @Test
+        fun `16-10 submission when system has submission base path`() {
```

Comment:
I'd suggest: "submission when the system has the basePath property configured"

However, I'd expect this property to be used later in subsequent tests which I think would make this particular test case obsolete